dev/com.ibm.ws.kernel.boot/publish/templates/servers/webProfile9/server.xml

```diff
@@ -3,7 +3,7 @@
 
     <!-- Enable features -->
     <featureManager>
-        <feature>microProfile-1.4</feature>
+        <feature>webProfile-9.0</feature>
```

Comment:
This should have been 9.1 but the webProfile9 zip isn't needed until GA so can fix it later.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/17603

Reply:
But why even have it be 9.0?  If you just go with the packaging to only have 9.1 in the webprofile package as discussed in other review comments, this file will need to change.